lib/openscap_parser/value_definition.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+module OpenscapParser
+  class ValueDefinition < XmlNode
+    include OpenscapParser::Util
+
+    def id
+      @id ||= parsed_xml['id']
+    end
+
+    def description
+      @description ||= newline_to_whitespace(
+        parsed_xml.at_css('description') &&
+          parsed_xml.at_css('description').text || ''
+      )
+    end
+
+    def title
+      @title ||= parsed_xml.at_css('title') &&
+        parsed_xml.at_css('title').text
+    end
+
+    def type
+      @type ||= parsed_xml['type'] || 'string'
+    end
+
+    def lower_bound
+      lower_bound_element = parsed_xml.at_xpath("lower-bound[@selector='']") || parsed_xml.at_xpath('lower-bound[not(@selector)]')
+      @lower_bound ||= lower_bound_element&.text
+    end
+
+    def upper_bound
+      upper_bound_element = parsed_xml.at_xpath("upper-bound[@selector='']") || parsed_xml.at_xpath('upper-bound[not(@selector)]')
+      @upper_bound ||= upper_bound_element&.text
+    end
+
+    def default_value
+      # The default value is the value or complex-value element with a empty or absent @selector
+      # If there is no value or complex-value element with an empty or absent @selector, the first value in
+      # the top down processing shall be the default element
+      # There should only be at most 1 value or complex value with an empty or absent @selector
+      value_element = parsed_xml.at_xpath("value[@selector='']") || parsed_xml.at_xpath('value[not(@selector)]') ||
+        parsed_xml.at_xpath("complex-value[@selector='']") || parsed_xml.at_xpath('complex-value[not(@selector)]') ||
+        parsed_xml.xpath("value | complex-value")[0]
+      @default_value ||= value_element&.text
+    end
+
+    def non_default_values
+      ndv = parsed_xml.xpath("value | complex-value").select { |v| v&.text != default_value }
+      @non_default_values ||= ndv.each_with_object({}) do |element, rv|
+        rv[element.at_xpath('@selector')&.text] = element&.text
+      end
+    end
```

Comment:
This needs some explanation...what is this and why is it needed?

Reply:
I added this comment but I don't think it was showing because I updated a small part and the file was outdated: 
```
This will give us all of the non-default values that can be included in a Value definition.
The keys of this hash will be the @selector attribute value for the non-default values. The values of this hash will be the non-default value that goes with that @selector attribute.
We'll need to import these non-default values and save their selector attributes in compliance-backend because the profiles under the benchmark can choose to use any of these values instead of the default value and the selector is how we'll know which value the profile wants to use.
```

When we're getting refined values for a profile, we can only get the `idref` and `selector` which we can use to find the actual `value`
`<refine-value idref="xccdf_org.ssgproject.content_value_var_selinux_policy_name" selector="targeted"/>`

This is why I think this method is necessary because it maps the `selectors` to their actual `values` for all of the non-default options in the Value definition.